http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeServiceDbQueries.scala

```diff
@@ -252,45 +221,17 @@ object RuntimeServiceDbQueries {
     }.toSeq
   }
 
-  def listRuntimes(labelMap: LabelMap,
```

Comment:
I redid these two queries (listRuntimeIdsForCreator and listRuntimes) in Slick, and removed listRuntimesForWorkspace as it was a subset of listRuntimes